pkg/clusterresource/builder.go

```diff
@@ -101,6 +101,12 @@ type Builder struct {
 	// ReleaseImage is a specific OpenShift release image to install this cluster with. Will override
 	// ImageSet.
 	ReleaseImage string
+
+	// MachineNetwork is the subnet to use for the cluster's machine network.
+	MachineNetwork string
+
+	// SkipMachinePoolGeneration is set to skip generating MachinePool objects
+	SkipMachinePoolGeneration bool
```

Comment:
What's the use case for this option. I can't immediately think of why we'd want to skip the machine pools.

Reply:
I put it there b/c our current OpenStack support has no support for MachinePools